_posts/2023-10-09-september-rubygems-updates.md

```diff
@@ -0,0 +1,76 @@
+
+In RubyGems, we updated the SPDX license list from 2023-04-28,  2023-06-18, to 2023-01-26, raised `Gem::Package::FormatError` when gem encounters corrupt EOF [#6882](https://github.com/rubygems/rubygems/pull/6882), and ensured that loading multiple gemspecs with legacy YAML class references does not warn [#6889](https://github.com/rubygems/rubygems/pull/6889).
+
+Some other improvements that landed into our repo this month but that are not included in the above releases are:
+
+- added `universal-java-19` to CI test setup Gemfile locks - [#6942](https://github.com/rubygems/rubygems/pull/6942).
+- fixed false positive SymlinkError in symbolic link directory - [#6947](https://github.com/rubygems/rubygems/pull/6947).
+- added support for the `ruby-3.2.2` format in the `ruby file:` Gemfile directive, and explicitly test the `3.2.2@gemset` format as rejected - [#6954](https://github.com/rubygems/rubygems/pull/6954).
+- reduced allocations for stub specifications - [#6972](https://github.com/rubygems/rubygems/pull/6972).
+- allowed standalone mode to work on a Windows edge case - [#6989](https://github.com/rubygems/rubygems/pull/6989).
+- improved release scripts - [#6999](https://github.com/rubygems/rubygems/pull/6999).
+- fixed safe marshal test on jruby - [#6984](https://github.com/rubygems/rubygems/pull/6984).
+
+In September, RubyGems gained [114 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2023-09-01%7D...master@%7B2023-09-30%7D) contributed by 14 authors. There were 2,455 additions and 571 deletions across 105 files.
+
+## RubyGems.org News
+
+This month, RubyGems.org saw several bug fixes and updates, some of which include:
+
+- added a log in Pusher when notify is called - [#4072](https://github.com/rubygems/rubygems.org/pull/4072).
+- added versions index on `lower(gem_full_name)` - [#4095](https://github.com/rubygems/rubygems.org/pull/4095).
+- added backfill for `spec_sha256` on versions - [#4083](https://github.com/rubygems/rubygems.org/pull/4083).
+- handled nil `api_key` in dashboards controller -[#4081](https://github.com/rubygems/rubygems.org/pull/4081).
+- precompile assets on CI before running tests - [#4059](https://github.com/rubygems/rubygems.org/pull/4059).
+- made all texts in about page translatable. - [#4063](https://github.com/rubygems/rubygems.org/pull/4063).
+- only validate version metadata on create/change - [#4100](https://github.com/rubygems/rubygems.org/pull/4100).
+- updated RubyGems & Bundler - [#4103](https://github.com/rubygems/rubygems.org/pull/4103).
+
+In September, RubyGems gained [64 new commits](https://github.com/rubygems/rubygems.org/compare/master@%7B2023-09-01%7D...master@%7B2023-09-30%7D) contributed by 5 authors. There were 1,855 additions and 1,070 deletions across 90 files.
+
+## Maintainer's Note
```

Comment:
Hey @indirect, I added a section highlighting the work the team did the past month, is this a good title or do you have something else in mind?

Reply:
I think in this case I would move these two items into the RubyGems section, ahead of the list of PRs. In the future, I would love to have a longer note that's more like a blog post, explaining why the work during a given month was important, and/or how it will help developers in the future, but I think we aren't quite there yet. :)

Reply:
Alright! Thanks for clarifying.